go/vt/vtgate/engine/route.go

```diff
@@ -67,6 +67,9 @@ type Route struct {
 	// in the final result. Rest of the columns are truncated
 	// from the result received. If 0, no truncation happens.
 	TruncateColumnCount int
+
+	// QueryTimeout
```

Comment:
How about "QueryTimeout contains the optional timeout (in milliseconds) to apply to this query